frontend/test/BackendSpec.purs

```diff
@@ -135,6 +138,34 @@ spec = describe "Backend" do
         Right Forbidden → pure unit
         result → fail $ "Forbidden expected, got: " <> show result
 
+  describe "List users" do
+    it "sends proper data to the backend" do
+      let
+        server ∷ Backend.Transport
+        server { method, responseFormat, headers } = do
+          method `shouldEqual` Left GET
+          responseFormat `assertResponseFormat` ResponseFormat.json
+          headers `shouldEqual`
+            [ AX.RequestHeader "Authorization"
+                ("Bearer " <> Token.toString token)
+            ]
+          respond ok200 { body = encodeJson mockUserList }
+      listUsersWithConfig server token >>= case _ of
+        Left err → fail $ show err
+        Right _srb → pure unit
```

Comment:
It would be nice to verify that `Right` side returns the `mockUserList`

Reply:
done!